### ISSUE TITLE
lib not found

### ISSUE THREAD

**Issue by launchpadinteractive** [benmerckx/gulp-haxe#2]
Knowing that this project is less than 30 days old and also given that most the key features that I need for compiling to JS are working great for me, I wanted to make a note that when trying to include a library like "jQueryExtern" using both and external build.hxml file or manually including the lib in the gulp file, I get an error during compile time. Stating that I do not have the library installed when I actually do. Any thoughts on why this error is being thrown? 

My gulpfile is as follows. 

	
~~~~
const gulp = require('gulp');
const haxe = require('gulp-haxe');

gulp.task('default', function() {

	return haxe({
		js: 'build/main.js',
		main: 'Test',
		cp: 'src',
		lib: 'jQueryExtern',
	})
	.pipe(gulp.dest(''));

});
~~~~

**Comment by benmerckx** [comment]
What OS are you using it on? Currently it just doesn't handle library management. On my system (linux) it seems to default to the 'global' haxelib, which is why it can find the libs. The haxe npm lib includes a haxelib binary but I'm not sure if it can be leveraged while compiling on all systems, so I'll have to look into it. Ideally the libs can be installed and used locally per project, and running the gulp task would do that for you.

**Comment by benmerckx** [comment]
My previous comment seems to be incorrect :) I simply had not tested this with any lib. I get the same error, but it's encouraging as this should allow all libs to be installed through the local haxelib.

**Comment by launchpadinteractive** [comment]
Hey @benmerckx thanks for the response on this. I am using a Mac. I was able to set and install the haxelib "jQueryExtern" locally using "haxelib newrepo" then "haxelib install jQueryExtern" in the gulpfile.js directory and everything worked great. I am now able to compile using both the .hxml file and using the lib: in the gulpfile with jQuery. Thanks!

**Comment by benmerckx** [comment]
I've included lib support in 0.1.0. It should automatically install libs for you to the local haxelib directory.

**Comment by launchpadinteractive** [comment]
Yep it does @benmerckx I was able to switch back to the default "haxelib --global", run gulp and everything works great! Thanks for the update!
